packages/website/ts/components/wallet/wallet.tsx

```diff
@@ -491,16 +508,56 @@ export class Wallet extends React.Component<WalletProps, WalletState> {
         if (_.isEmpty(tokenAddresses)) {
             return {};
         }
+        // for each input token address, search for the corresponding symbol in this.props.tokenByAddress, if it exists
+        // create a mapping from existing symbols -> address
+        const tokenAddressBySymbol = _.fromPairs(
+            _.compact(
+                _.map(tokenAddresses, address => {
+                    const tokenIfExists = _.get(this.props.tokenByAddress, address);
+                    if (!_.isUndefined(tokenIfExists)) {
+                        const symbol = tokenIfExists.symbol;
+                        // the crypto compare api doesn't understand 'WETH' so we need to replace it with 'ETH'
+                        const key = symbol === ETHER_TOKEN_SYMBOL ? ETHER_SYMBOL : symbol;
+                        return [key, address];
+                    } else {
+                        return undefined;
+                    }
+                }),
+            ),
+        );
+        const joinedTokenSymbols = _.keys(tokenAddressBySymbol).join(',');
+        const quoteCurrency = 'USD';
```

Comment:
Let's move this to `config.ts`. I could see us making this configurable at a later date and it would be best to only store this setting in one place.